src/app/payments/send-intraledger.ts

```diff
@@ -376,6 +380,21 @@ const executePaymentViaIntraledger = async <
       await removeDeviceTokens({ userId: recipientUser.id, deviceTokens: result.tokens })
     }
 
+    if (
+      recipientAccount.level === AccountLevel.One ||
+      recipientAccount.level === AccountLevel.Two
+    ) {
+      const callbackService = CallbackService(getCallbackServiceConfig())
+      await callbackService.sendMessage({
+        accountUUID: recipientAccount.uuid,
+        eventType: CallbackEventType.ReceiveIntraledger,
```

Comment:
having this makes me wonder if we should use notification service as a wrapper of svix/callbacks :thinking:

Reply:
You mean notifications should be an endpoint from this callback?

I think that would make sense